client/my-sites/patterns/index.tsx

```diff
@@ -0,0 +1,33 @@
+import DocumentHead from 'calypso/components/data/document-head';
+import Main from 'calypso/components/main';
+import { PatternsCategoryBody } from 'calypso/my-sites/patterns/bodies/category';
+import { PatternsHomeBody } from 'calypso/my-sites/patterns/bodies/home';
+import type { PatternGalleryFC } from 'calypso/my-sites/patterns/types';
+
+type PatternsHomePageProps = {
+	category: string;
+	isGridView?: boolean;
+	patternGallery: PatternGalleryFC;
+};
+
+export const PatternsIndex = ( {
+	category,
+	isGridView,
+	patternGallery: PatternGallery,
+}: PatternsHomePageProps ) => {
+	return (
+		<Main isLoggedOut fullWidthLayout>
+			<DocumentHead title="WordPress Patterns" />
```

Comment:
We will most certainly have different titles for each page, for SEO reasons. So this should be moved (and duplicated) in each subcomponents.

Reply:
And [a footer](https://github.com/Automattic/wp-calypso/pull/88245) soon 😀.